### ISSUE TITLE
DenseNet low accuracy

### ISSUE THREAD

**Issue by mongoose54** [tdeboissiere/DeepLearningImplementations#30]
@tdeboissiere Just wanted to tell you excellent repository.

I am running the  `run_cifar10.py` example on the DenseNet architecture but the accuracy is low. Do you get similar results? Please see the output:

Epoch 1/30, Time: 51.1712930202
Validation loss:  1.9541955759   accuracy:  0.3231
Epoch 2/30, Time: 48.110555172
Validation loss:  1.8552568697   accuracy:  0.3467
Epoch 3/30, Time: 48.1387000084
Validation loss:  1.80415105343   accuracy:  0.3602
Epoch 4/30, Time: 48.2201020718
Validation loss:  1.76553211613   accuracy:  0.3693
Epoch 5/30, Time: 48.2158961296
Validation loss:  1.7347150177   accuracy:  0.3793
Epoch 6/30, Time: 48.209141016
Validation loss:  1.70514687557   accuracy:  0.388
Epoch 7/30, Time: 48.237156868
Validation loss:  1.67561238804   accuracy:  0.3969
Epoch 8/30, Time: 48.1645870209
Validation loss:  1.65338623161   accuracy:  0.4058
Epoch 9/30, Time: 48.2082428932
Validation loss:  1.6369724762   accuracy:  0.4104
Epoch 10/30, Time: 48.1478528976
Validation loss:  1.62344400311   accuracy:  0.4158
Epoch 11/30, Time: 48.1710720062
Validation loss:  1.61286505413   accuracy:  0.4184
Epoch 12/30, Time: 48.1577467918
Validation loss:  1.6039566658   accuracy:  0.4221
Epoch 13/30, Time: 48.1328320503
Validation loss:  1.59814694672   accuracy:  0.4249
Epoch 14/30, Time: 48.1533429623
Validation loss:  1.59282479839   accuracy:  0.427
Epoch 15/30, Time: 48.1005380154
Validation loss:  1.58933777313   accuracy:  0.4292
Epoch 16/30, Time: 48.1664721966
Validation loss:  1.55802340374   accuracy:  0.4383
Epoch 17/30, Time: 48.1234638691
Validation loss:  1.55658948803   accuracy:  0.4395
Epoch 18/30, Time: 48.195486784
Validation loss:  1.55546182899   accuracy:  0.4394
Epoch 19/30, Time: 48.1643540859
Validation loss:  1.55441089878   accuracy:  0.4399
Epoch 20/30, Time: 48.1657791138
Validation loss:  1.5535571804   accuracy:  0.4409
Epoch 21/30, Time: 48.129434824
Validation loss:  1.55240019016   accuracy:  0.4407
Epoch 22/30, Time: 48.1452748775
Validation loss:  1.55181030369   accuracy:  0.4411
Epoch 23/30, Time: 48.1575331688
Validation loss:  1.55042634773   accuracy:  0.4435
Epoch 24/30, Time: 48.1651659012
Validation loss:  1.55020442104   accuracy:  0.4435
Epoch 25/30, Time: 48.1427268982
Validation loss:  1.55003917351   accuracy:  0.4442
Epoch 26/30, Time: 48.1740958691
Validation loss:  1.54997017498   accuracy:  0.4446
Epoch 27/30, Time: 48.1708450317
Validation loss:  1.54985166111   accuracy:  0.4442
Epoch 28/30, Time: 48.1398389339
Validation loss:  1.54980517941   accuracy:  0.444
Epoch 29/30, Time: 48.1378748417
Validation loss:  1.54972298145   accuracy:  0.4439
Epoch 30/30, Time: 48.1508190632
Validation loss:  1.54960639267   accuracy:  0.444

**Comment by tdeboissiere** [comment]
Make sure you are annealing the learning rate correctly and training long enough.
Does your training accuracy increase normally ?
